File: Source/ecs/Source/Library/philipsecommercesdk/src/main/java/com/philips/platform/ecs/model/retailers/OnlineStoresForProductEntity.java
/* Copyright (c) Koninklijke Philips N.V., 2018
 * All rights are reserved. Reproduction or dissemination
 * in whole or in part is prohibited without the prior written
 * consent of the copyright holder.
 */
package com.philips.platform.ecs.model.retailers;

import java.io.Serializable;

public class OnlineStoresForProductEntity implements Serializable {
    private String excludePhilipsShopInWTB;
    private String showPrice;
    private String ctn;
    private ECSRetailers Stores;

    public String getExcludePhilipsShopInWTB() {
        return excludePhilipsShopInWTB;
    }

    public String getShowPrice() {
        return showPrice;
    }

    public String getCtn() {
        return ctn;
    }

    public ECSRetailers getStores() {
        return Stores;
    }
}

File: Source/ecs/Source/Library/philipsecommercesdk/src/main/java/com/philips/platform/ecs/microService/model/retailers/OnlineStoresForProductEntity.kt
/*
 *  Copyright (c) Koninklijke Philips N.V., 2020
 *
 *  * All rights are reserved. Reproduction or dissemination
 *
 *  * in whole or in part is prohibited without the prior written
 *
 *  * consent of the copyright holder.
 *
 *
 */
package com.philips.platform.ecs.microService.model.retailers

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize
@Parcelize
data class OnlineStoresForProductEntity (
    val excludePhilipsShopInWTB: String?,
    val showPrice: String?,
    val ctn: String?,
    val stores: ECSRetailers?

):Parcelable